genai-perf/genai_perf/config/input/base_config.py

```diff
@@ -56,6 +57,9 @@ def any_field_set_by_user(self) -> bool:
 
     def check_required_fields_are_set(self) -> None:
         for name, field in self._fields.items():
+            # Skip validation model_names is set for process-export-files subcommand
```

Comment:
This isn't the right level to have this check. The BaseConfig class should have no knowledge about decisions we are making when creating config files (think of it as a library).
I think the proper course of action here is to remove the requirement of model name being set from ConfigCommand if this subcomand is set in the check_required_fields_are_set() in that file.

```
def _check_required_fields_are_set(self) -> None:
        self.get_field("model_name").required = False
        super().check_required_fields_are_set()
```